src/Core/Shipping/BulkSender.cs

```diff
@@ -36,11 +43,11 @@ public Task<HttpResponseMessage> SendAsync(ICollection<LogzioLoggingEvent> logz,
                 return Task.FromResult(new HttpResponseMessage(System.Net.HttpStatusCode.OK));
 
             var url = string.Format(UrlTemplate, options.ListenerUrl, options.Token, options.Type);
-            var body = SerializeLogEvents(logz, _encodingUtf8, options.UseGzip);
+            var body = SerializeLogEvents(logz, _encodingUtf8);
             return _httpClient.PostAsync(url, body, _encodingUtf8, options.UseGzip);
         }
 
-        private MemoryStream SerializeLogEvents(ICollection<LogzioLoggingEvent> logz, System.Text.Encoding encodingUtf8, bool useGzip)
+        private MemoryStream SerializeLogEvents(ICollection<LogzioLoggingEvent> logz, System.Text.Encoding encodingUtf8)
```

Comment:
@snakefoot Why did you drop gzip support?

Reply:
I just removed an unused parameter. The gzip support handling is not done here.

Reply:
Here the gzip-support is handled:

https://github.com/logzio/logzio-dotnet/blob/c9b5e6c67548b34ee32d820105550b3971cd7ebb/src/Core/HttpClient/IHttpClient.cs#L47-L57